docs/topics/review-existing-rules.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * docs/rules-development-guide/master.adoc
 
+:_content-type: REFERENCE
```

Comment:
@stoobie @sbeskin-redhat From title and contents, I think this is a procedure (though it should be rewritten to fit our  style).  It gives too much advice/instruction for the average user to be a reference module, in my opinion. What do you think?

Reply:
@RichardHoch  The title is indeed procedure, but the contents are not, in my view. Not only because it does not have Procedure and Prerequisites sections, but because it does not have any precise instructions. This module specifies the location of the rules and provides some recommendations on how to handle them. Looks more like reference to me.

Reply:
@sbeskin-redhat @RichardHoch 

This section and the next appear to me to be one procedure module. This file is the introduction to the procedure. What is currently the title `Forking and cloning the Migration Toolkit for Applications XML rules` should be `.Procedure`.
`You must have the git client installed on your machine.` is a prerequisite.